### PR TITLE
chore(open-ux-tools) Update npmrc file for more reliable install and lock file updates. Update devDeps

### PR DESCRIPTION
**Pull request by devinea**
chore(open-ux-tools) Update npmrc file for more reliable install and lock file updates. 
Update devDeps

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Override users global npmrc
+registry = https://registry.npmjs.org/
+@sap-ux:registry = https://registry.npmjs.org/
+@sap:registry = https://registry.npmjs.org/
+@sapux:registry = https://registry.npmjs.org/
+@types:registry = https://registry.npmjs.org/
+
+audit = true
+package-lock = true
```
Comment (donal-tobin-sap) [inline .npmrc:9]: Not used because you do not use npm to install.

File: .npmrc [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+strict-ssl = true
+scripts-prepend-node-path = true
+fund = false
```
Comment (donal-tobin-sap) [inline .npmrc:12]: Not very nice to the open source community

**Review by donal-tobin-sap — DISMISSED** [review]
👍🏽 Looks good to me.

File: .npmrc (hunk 1 above)
Comment (devinea) [reply .npmrc:9]: pnpm seems to respect this as I had it set to false initially and the lock file was not updated and getting a warninn in the console

**Review by donal-tobin-sap — APPROVED** [review]

**Review by kranthie-sap — APPROVED** [review]
Looks good to me. Thanks.

**Pull request merged**